app/models/spree/image.rb

```diff
@@ -9,6 +9,11 @@ class Image < Asset
       large: { resize_to_limit: [600, 600] },
     }.freeze
 
+    self.ignored_columns = %i(attachment_file_name
+                              attachment_content_type
+                              attachment_file_size
+                              attachment_updated_at)
```

Comment:
I also found `attachment_width` and `attachment_height` which should be ignored and later deleted.

Reply:
Sure, @mkllnk I will add them to this list

Reply:
Done, added both the columns to the ignored list @mkllnk